### ISSUE TITLE
Segfault with derep_id on both strands

### ISSUE THREAD

**Issue by torognes** [torognes/vsearch#565]
Frédéric discovered that vsearch 2.28.1 crashes with a segmentation fault when running `derep_id` on both strands.

It can be reproduced like this:

```sh
printf ">s1;size=1;\nA\n>s1;size=1;\nT\n" | \
   vsearch \
       --derep_id - \
       --minseqlength 1 \
       --strand both \
       --quiet \
       --output /dev/null
```

**Comment by torognes** [comment]
Fixed in commit a05b2ea in dev branch.

**Comment by frederic-mahe** [comment]
test is now part of our test-suite (https://github.com/frederic-mahe/vsearch-tests/commit/b4f18d54effd936786a6d21be4a3f5b561aa705d)
